src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card"
+         data-qa="card"
+    >
+      <div class="card__foto_card">
+        <img class="card__foto_card_position"
+             src="/images/imac.jpeg"
+             alt="imac"
+        >
+      </div>
+
+      <div class="card__product_info">
+        <h2 class="card__product_name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+        <div class="card__product_code">
+          Код товара: 195434
+        </div>
+        <div class="card__product_rating">
+          <div class="card__stars--active__stars stars">
```

Comment:
```suggestion
          <div class="card__stars stars stars--4">
```
Тут должен быть такой модификатор, который отвечает за количество активных звездочек